### DISCUSSION TITLE
[Bug]: name issue in preview

### DISCUSSION THREAD

**Discussion by VE7OES** [meshtastic/firmware#4852] [Troubleshooting]
### Category

Other

### Hardware

T-Beam

### Firmware Version

2.5.0.9

### Description

When a t-beam is upgraded to the preview release, the long and short name are fine. If there's a full reset done to the t-beam, attempting to change the long and/or short name doesn't take. The t-beam reboots with the default names. Every-time.

**Comment by thebentern** [comment]
If you're doing this via one of the apps, are you able to grab some serial logs via `meshtastic --noproto` or the serial monitor https://flasher.meshstastic.org?

**Comment by jp-bennett** [comment]
Which app are you using to try to set the names? And what version of that app, if you can find it?
